docs/react/api-workspace/schematics/workspace-schematic.md

```diff
@@ -6,7 +6,20 @@ Generates a workspace schematic
 
 ```bash
 nx generate workspace-schematic ...
+```
+
+By default, Nx will search for `workspace-schematic` in the default collection provisioned in `workspace.json`.
+
+You can specify the collection explicitly as follows:
+
+```bash
+nx g @nrwl/workspace:workspace-schematic ...
+```
 
+Show what will be generated without writing to disk:
+
+```bash
+nx g null ... --dry-run
```

Comment:
There is still null here. This should be ng g workspace-schematic or ng g @nrwl/workspace:workspace-schematic --dry-run